secretstores/hashicorp/vault/vault.go

```diff
@@ -508,5 +508,9 @@ func readCertificateFolder(certPool *x509.CertPool, path string) error {
 
 // Features returns the features available in this secret store.
 func (v *vaultSecretStore) Features() []secretstores.Feature {
-	return []secretstores.Feature{secretstores.FeatureMultipleKeyValuesPerSecret}
+	if v.vaultValueType == valueTypeText {
+		return []secretstores.Feature{}
+	} else {
+		return []secretstores.Feature{secretstores.FeatureMultipleKeyValuesPerSecret}
+	}
```

Comment:
nit: omitting the else and return straight is part of the "go philosophy" 

[source](https://go.dev/doc/effective_go) 
<img width="1214" alt="image" src="https://user-images.githubusercontent.com/5839364/204395613-e48a8b8b-1b1b-41e0-bc85-f60840593a27.png">

Reply:
Cool! Updated! Thanks :)